cmd/charmd/main.go

```diff
@@ -92,3 +92,16 @@ func serve(confPath string) error {
 	logger.Infof("starting the API server")
 	return http.ListenAndServe(conf.APIAddr, debug.Handler("", server))
 }
+
+var mgoLogger = loggo.GetLogger("mgo")
+
+func init() {
+	mgo.SetLogger(mgoLog{})
+}
+
+type mgoLog struct{}
+
+func (mgoLog) Output(calldepth int, s string) error {
+	mgoLogger.LogCallf(calldepth+1, loggo.INFO, "%s", s)
```

Comment:
Should we put this on the same level as whole charmd is run? Or is INFO enough here?